code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -632,6 +632,13 @@
 		M.adjust_nutrition(-3*nutriment_factor)
 	..()
 
+/datum/reagents/consumable/bakingsoda
+  name = "Baking Soda"
```

Comment:
that's a funny lookin' tab you've got there

Reply:
The funny part is that I only touched the stuff I wrote :v

Reply:
Also 
![image](https://user-images.githubusercontent.com/9438930/50389021-22f75e80-06f2-11e9-9957-5ab50178fa17.png)

Reply:
Found the issue, resolving